assets/js/googlesitekit/components-gm2/Checkbox.js

```diff
@@ -44,13 +44,15 @@ export default function Checkbox( props ) {
 		tabIndex,
 		onKeyDown,
 		loading,
+		alignLeft,
```

Comment:
Do we also need to apply this to the gm3 checkbox?

Reply:
Applied, thank you!